<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMovieMakerPreferencePage</class>
 <widget class="QWidget" name="QmitkMovieMakerPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>133</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ffmpegLabel">
     <property name="text">
      <string>FFmpeg path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="ffmpegLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ffmpegButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Video output format:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="vp9Button">
       <property name="text">
        <string>VP9 (*.webm)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">formatButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="h264Button">
       <property name="text">
        <string>H.264 (*.mp4)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">formatButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="formatButtonGroup"/>
 </buttongroups>
</ui>
